ugs-platform/ugs-platform-ugslib/src/main/java/com/willwinder/ugs/nbp/lib/services/ActionRegistrationService.java

```diff
@@ -144,4 +155,87 @@ public void overrideActionName(String category, String key, String name) {
             Exceptions.printStackTrace(ex);
         }
     }
+
```

Comment:
Nice! The NetBeans people might like to hear about this: https://netbeans.apache.org/wiki/DevFaqActionsAddAtRuntime.asciidoc